t/t1019-mf-priority-info-fetch.t

```diff
@@ -62,8 +61,8 @@ test_expect_success 'add some projects to the DB' '
 '
 
 test_expect_success 'add a user with two different banks to the DB' '
-	flux account add-user --username=user1001 --userid=1001 --bank=account1 --max-running-jobs=2 &&
-	flux account add-user --username=user1001 --userid=1001 --bank=account2
+	flux account add-user --username=user5001 --userid=5001 --bank=account1 --max-running-jobs=2 &&
+	flux account add-user --username=user5001 --userid=5001 --bank=account2
```

Comment:
Is there something in the testsuite that ensures the user running it is not 5001 (or for that matter is 1001). Or is the testsuite only expected to be run in certain environments?

One idea would be to add a check for the current uid in `sharness.d/flux-accounting.sh` and set up some environment variables with userids instead of explicitly using 1001 and 5001 (in case those conflict with system uids)

This could be done in a future cleanup PR if this is working for now.

Reply:
I guess there isn't anything explicit that is checking it (I just checked the logs from the tests that failed via GitHub actions), but that's a good idea to set up some environment variables so that I don't have to define them every single time I create a new test. I'll open an issue on this so I can get this cleaned up in the future.